src/hardware/introduction.md

```diff
@@ -8,7 +8,7 @@ permalink: /hardware/introduction/
 
 Geotab supports Hardware Add-On integration through the CAN, USB, RS232, and BLE interfaces. The Add-On Protocols section outlines the integration process for all four interfaces. Geotab's open platform allows for expandability to support the unique data sets of any Hardware Add-On. The GO device can currently report Hardware Add-On Data Types found on [MyGeotab Diagnostics](https://docs.google.com/spreadsheets/d/1sy7IOhWmFoo40_I-ruOJO8bVTMLXqHa11d0oJtaoIcE/edit#gid=1133172080).
 
-When using Geotab's Hardware Add-On protocols, it is important not to send excessive amounts of data to the GO device.  This can be assured by only sending data when it is completely necessary and avoiding the sending of repetitive values. The GO device will stop recording data from the Hardware Add-On device if excessive Add-On data is detected. When this condition occurs, "Status data no longer being saved due to excessive data" will appear in the Fault Data for the GO device within MyGeotab. In order to re-enable data logging for the Hardware Add-On, the GO device will need to be completely disconnected from power and then powered back up again.
+When using Geotab's Hardware Add-On protocols, it is important not to send excessive amounts of data to the GO device.  This can be assured by only sending data when it is completely necessary and avoiding the sending of repetitive values. The GO device will stop recording data from the Hardware Add-On device if excessive Add-On data is detected. When this condition occurs, "Status data no longer being saved due to excessive data" or "Third-party data no longer being saved due to excessive data" will appear in the Fault Data for the GO device within MyGeotab. In order to re-enable data logging for the Hardware Add-On, the GO device will need to be completely disconnected from power and then powered back up again.
```

Comment:
There is also a FAULT_BLUETOOTH_DATA_LOGGING_TURNED_OFF.

Reply:
Hi @Stephen-F , no entry in emGoPid matches FAULT_BLUETOOTH_DATA_LOGGING_TURNED_OFF, code 376. That is a mistake, correct? I can help to correct it. Would you like customers to see "Bluetooth data no longer being saved due to excessive data" on the UI?